### PR TITLE
setup: Move to venv

### PR DESCRIPTION
**Pull request by benmcmahon100**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,28 +10,38 @@ Available at [docs.redbrick.dcu.ie](https://docs.redbrick.dcu.ie) and at the
...
+mkdocs serve
+```
 
-```bash
-docker-compose run --rm docs mkdocs build
-docker-compose run --rm test
+To test the docs build and if there are any dead links in them, use this:
+
+```sh
+mkdocs build
```
Comment (m1cr0man) [inline README.md]: Can you add -s and -v please

File: README.md [hunk 2]
```diff
@@ -10,28 +10,38 @@ Available at [docs.redbrick.dcu.ie](https://docs.redbrick.dcu.ie) and at the
...
-docker-compose up docs
-```
+When you make a change make sure it works by building it locally.
 
-To build the docs just run
+Run:
 
 ```bash
-docker-compose up -d docs
+python3 -m venv .venv
```
Comment (m1cr0man) [inline README.md:17]: Can you add .venv to .gitignore?

**Review by m1cr0man — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (benmcmahon100) [reply README.md]: Yessie, was just about to at seeing you PR

File: README.md (hunk 2 above)
Comment (benmcmahon100) [reply README.md:17]: it's already there - that's why I used the `.`

**Review by m1cr0man — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -10,28 +10,38 @@ Available at [docs.redbrick.dcu.ie](https://docs.redbrick.dcu.ie) and at the
...
+.venv/bin/activate
+pip install -r requirements.txt
 ```
 
-## Testing locally
+For fish users:
+```fish
+python3 -m venv .venv
+chmod +x -R .venv/bin
+. .venv/bin/activate
```
Comment (butlerx) [inline README.md:19]: this works in bash also.

**Review by butlerx — CHANGES_REQUESTED** [review]

**Review by butlerx — DISMISSED** [review]

**Review by butlerx — APPROVED** [review]

**Pull request merged**